runtime/queries/gomod/injections.scm

```diff
@@ -0,0 +1 @@
+(comment) @comment
```

Comment:
Injections for helix are a different format than nvim-treesitter - we follow the [tree-sitter way](https://tree-sitter.github.io/tree-sitter/syntax-highlighting#language-injection)

So the comment injections come out to:

```scm
((comment) @injection.content
 (#set! injection.langauge "comment"))
```